src/tests/EdFi.DataManagementService.Tests.E2E/Features/SuperclassReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,120 @@
+Feature: SuperclassReferenceValidation of Creation, Update and Deletion of resources
+
+        Background:
+            Given this Local Education Agency exists
+                  | localEducationAgencyId | nameOfInstitution           | localEducationAgencyCategoryDescriptor | educationOrganizationCategoryDescriptor |
+                  | 101                    | Local Education Agency Test | ["Other local education agency"]       | ["Local Education Agency"]              |
+              And this School exists
+                  | schoolId | nameOfInstitution | localEducationAgencyId | educationOrganizationCategoryDescriptor | gradeLevelDescriptor |
+                  | 100      | School Test       | 101                    | ["School"]                              | ["Ninth grade"]      |
+
+        @ignore
+        Scenario: Ensure clients can create a Program that references an existing School
```

Comment:
Please add numbers for each scenario, following the pattern in PR #157.

Reply:
Added